blocks/layout-grid/src/grid.scss

```diff
@@ -18,7 +18,7 @@
 
 	// Every block, including nested blocks, are born with an intrinsic minimum margin.
 	// This resets that for the containers, which aren't meant to have that intrinsically.
-	> .wp-block > .editor-block-list__block-edit > [data-block] {
+	> .wp-block > .block-editor-block-list__block-edit > [data-block] {
```

Comment:
I think this is dead CSS, but I can't be sure. I searched the DOM for `wpcom-resize-grid` but can't find it. What's it used for?

Reply:
I think it is dead, yes